guides/v2.1/magento-devdocs-whatsnew.md

```diff
@@ -9,9 +9,14 @@ We exclude from this list proofreading, spelling checks, and all minor updates.
 
 ## October 2018
 
-Description  | Versions applied to  | New or Updated | Date
--------------|--------------|----------------------|--------
-Added [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.2/security/two-factor-authentication.html) and [Google reCAPTCHA](https://devdocs.magento.com/guides/v2.2/security/google-recaptcha.html) installation and troubleshooting information for [Security settings](https://devdocs.magento.com/guides/v2.2/config-guide/secy/secy.html).|2.1.x, 2.2.x|New topic|October  1
+Description                                                                                                                                                                                                                                                                                                                                                                                                                           | Versions applied to | New or Updated    | Date
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|-------------------|-----------
+Added [Multi-Repo Docs](https://devdocs.magento.com/community/resources/multi-repo-docs.html) features and overview                                                                                                                                                                                                                                                                                                                   | 2.x                 | New topic         | October  7
+MFTF 2.3.7 release:<br/>Updated the [pressKey](https://devdocs.magento.com/mftf/2.3/test/actions.html#presskey) action.<br/>Updated the [action group argument](https://devdocs.magento.com/mftf/2.3/test/action-groups.html#data-type-usage) syntax for a value based on data entity resolution.<br/>Added the [DEFAULT_TIMEZONE](https://devdocs.magento.com/mftf/2.3/configuration.html#default_timezone) configuration parameter. | 2.3.x               | Technical changes | October  5
```

Comment:
Putting the updated items in a bulleted list would be more effective than line breaks.  I'm pretty sure you can do that--we have it in the [Cloud Architecture](https://devdocs.magento.com/guides/v2.2/cloud/architecture/cloud-architecture.html) article.  If bullets won't work, maybe dashes.

Reply:
It must be done in the corresponding PR, because the process is automated and the table contains only imports from the corresponding PRs.
The [table you mentioned](https://raw.githubusercontent.com/magento/devdocs/master/guides/v2.1/cloud/architecture/cloud-architecture.md) is in HTML. We're avoiding HTML tables for the reason of better readability and maintenance of the topic.
It is still possible to add simple `-` or `*`, though they won't be converted from Markdown to  HTML on the resulting webpage.